python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -526,24 +543,20 @@ class RandomForestClassifier(Base):
 
         return ctypes.c_void_p(mod_handle).value
 
-    def _concatenate_treelite_handle(self, treelite_handle):
+    def _concatenate_treelite_handles(self, treelite_handles):
         cdef ModelHandle concat_model_handle = NULL
         cdef vector[ModelHandle] *model_handles \
             = new vector[ModelHandle]()
         cdef uintptr_t mod_ptr
-        for i in treelite_handle:
+        for i in treelite_handles:
             mod_ptr = <uintptr_t>i
             model_handles.push_back((
                 <ModelHandle> mod_ptr))
 
         concat_model_handle = concatenate_trees(deref(model_handles))
         cdef uintptr_t concat_model_ptr = <uintptr_t> concat_model_handle
         self.treelite_handle = concat_model_ptr
-        cdef vector[unsigned char] pbuf_mod_info = \
```

Comment:
If we dont obtain the pbuf_bytes for the concatenated model then the self.model_pbuf_bytes never get updated since the `self.model_pbuf_bytes` of the non concatenated model are already present and on running `_getstate_` the old model_pbuf_bytes` are returned.